modules/git/blame.go

```diff
@@ -98,17 +105,32 @@ func (r *BlameReader) Close() error {
 	r.bufferedReader = nil
 	_ = r.reader.Close()
 	_ = r.output.Close()
+	if r.ignoreRevsFile != nil {
+		_ = util.Remove(r.ignoreRevsFile.Name())
+	}
 	return err
 }
 
 // CreateBlameReader creates reader for given repository, commit and file
-func CreateBlameReader(ctx context.Context, repoPath, commitID, file string) (*BlameReader, error) {
-	cmd := NewCommandContextNoGlobals(ctx, "blame", "--porcelain").
-		AddDynamicArguments(commitID).
+func CreateBlameReader(ctx context.Context, repoPath string, commit *Commit, file string, bypassBlameIgnore bool) (*BlameReader, error) {
+	cmd := NewCommandContextNoGlobals(ctx, "blame", "--porcelain")
+
+	var ignoreRevsFile *os.File
```

Comment:
`As of git 2.23, git-blame supports ignoring specific commits.`
https://github.com/postgres/postgres/blob/master/.git-blame-ignore-revs.
Version check missing.